examples/experimental/dagster-dlift/dagster_dlift/cloud_instance.py

```diff
@@ -139,3 +140,26 @@ def get_dbt_sources(self, environment_id: int) -> Sequence[Mapping[str, Any]]:
                 "endCursor"
             ]
         return sources
+
+    def get_dbt_tests(self, environment_id: int) -> Sequence[Mapping[str, Any]]:
```

Comment:
similar to comment on stacked pr - could we pull out shared pagination logic to make this impl even simpler?